.gitattributes

```diff
@@ -0,0 +1,2 @@
+spec/fixtures/vcr_cassettes -diff
```

Comment:
sadly, it doesn't look like this affected the diff showing here in github "files changed"